polytope/polytope.py

```diff
@@ -772,21 +775,22 @@ def is_convex(reg, abs_tol=ABS_TOL):
     outer = envelope(reg)
     if is_empty(outer):
         # Probably because input polytopes were so small and ugly..
-        return False,None
+        return False, None
 
-    Pl,Pu = reg.bounding_box
-    Ol,Ou = outer.bounding_box
+    Pl, Pu = reg.bounding_box
+    Ol, Ou = outer.bounding_box
 
-    bboxP = np.hstack([Pl,Pu])
-    bboxO = np.hstack([Ol,Ou])
+    bboxP = np.hstack([Pl, Pu])
+    bboxO = np.hstack([Ol, Ou])
 
-    if sum(abs(bboxP[:,0] - bboxO[:,0]) > abs_tol) > 0 or \
-    sum(abs(bboxP[:,1] - bboxO[:,1]) > abs_tol) > 0:
-        return False,None
+    if sum(abs(bboxP[:, 0] - bboxO[:, 0]) > abs_tol) > 0 or \
```

Comment:
@carterbox you can increase (or decrease up to 3 space chars) the indent of the `return` statement without error; cf. https://docs.python.org/3/reference/lexical_analysis.html#indentation
However, consistency is better kept if it stays where it is.

I support the extra indentation proposed by @johnyf. In this case of the first disjunct beginning with `sum(`, which is 4 char wide, there is some risk of confusing the second conjunct as part of the first `sum`.